test/libsyncengine/reconciliation/platform_inconsistency_checker/testplatforminconsistencycheckerworker.cpp

```diff
@@ -154,6 +165,78 @@ void TestPlatformInconsistencyCheckerWorker::testNameClash() {
 #endif
 }
 
+void TestPlatformInconsistencyCheckerWorker::testNameClashAfterRename() {
+    // Create local files
+    std::vector<std::string> nodes = {{"a1"}, {"A"}};
+    for (const auto &n : nodes) {
+        std::ofstream ofs(_tempDir.path / n);
+        ofs << "Some content.\n";
+        ofs.close();
+    }
+
+    // Set up DB
+    DbNode dbNode_A(2, _syncPal->_syncDb->rootNode().nodeId(), Str("A"), Str("A"), "lA", "rA", defaultTime, defaultTime,
+                    defaultTime, NodeType::NodeTypeDirectory, defaultSize, std::nullopt);
+    DbNode dbNode_a(3, _syncPal->_syncDb->rootNode().nodeId(), Str("a1"), Str("a1"), "la", "ra", defaultTime, defaultTime,
+                    defaultTime, NodeType::NodeTypeDirectory, defaultSize, std::nullopt);
+    DbNodeId dbNodeIdDir_A;
+    DbNodeId dbNodeIdDir_a;
+    bool constraintError = false;
+    _syncPal->_syncDb->insertNode(dbNode_A, dbNodeIdDir_A, constraintError);
+    _syncPal->_syncDb->insertNode(dbNode_a, dbNodeIdDir_a, constraintError);
+
+    // Set up remote tree
+    std::shared_ptr<Node> remoteParentNode = _syncPal->_remoteUpdateTree->rootNode();
```

Comment:
```suggestion
    std::shared_ptr<Node> remoteParentNode = _syncPal->getUpdateTree(ReplicaSideRemote)->rootNode();
```

Reply:
Is it `getUpdateTree` or simply `updateTree`?